src/ui/components/Errors/RootErrorDocumentTitle.tsx

```diff
@@ -0,0 +1,14 @@
+import Head from "next/head";
+
+import { useGetRecordingTitle } from "shared/graphql/Recording";
+import { getRecordingId } from "shared/utils/recording";
+
+export function RootErrorDocumentTitle() {
+  const { title = "Replay" } = useGetRecordingTitle(getRecordingId());
```

Comment:
One flaw of this is, I guess, if GraphQL goes down, our error boundary will also crash.